receiver/awsxrayreceiver/go.mod

```diff
@@ -5,13 +5,9 @@ go 1.14
 replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/common => ../../internal/common
 
 require (
-	github.com/aws/aws-sdk-go v1.34.5
+	github.com/aws/aws-sdk-go v1.34.8
 	github.com/google/uuid v1.1.1
-	github.com/googleapis/gnostic v0.4.1 // indirect
 	github.com/stretchr/testify v1.6.1
-	go.opentelemetry.io/collector v0.8.1-0.20200819173546-64befbcc0564
+	go.opentelemetry.io/collector v0.8.0
```

Comment:
@bogdandrutu Is this version right?

Reply:
No we should use the one that is there already and in all the targets